Assets/Scripts/Chat/ChatChannels.cs

```diff
@@ -12,4 +12,12 @@ public List<ChatChannel> GetChannels()
     {
         return Channels;
     }
+    public List<ChatChannel> GetUnhidable()
+    {
+        return Channels.FindAll(channel => {return channel.Unhidable;});
+    }
+    public List<ChatChannel> GetHidable()
+    {
+        return Channels.FindAll(channel => {return !channel.Unhidable;});
+    }

```

Comment:
I am not 100% sure on this, but won't an `IEnumerator<ChatChannel>` cause less garbage allocations?

Reply:
Resolved in voice chat